modules/nw-ingress-configuring-application-domain.adoc

```diff
@@ -5,7 +5,7 @@
 
 :_content-type: PROCEDURE
 [id="nw-ingress-configuring-application-domain_{context}"]
-= Configuring application domain for the Ingress Controller Operator
+= Configuring cluster domain for applications using appsdomain option.
```

Comment:
You could remove the full-stop from the end of this heading, for consistency with other headings in the library.

Reply:
Acked